.rubocop.yml

```diff
@@ -16,22 +16,39 @@
 
 LineLength:
   Max: 140
+
 MethodLength:
   Max: 40
+
+Metrics/AbcSize:
+  Max: 20
+
 Metrics/BlockLength:
   Enabled: false
-Style/AccessorMethodName:
-  Enabled: false
-Metrics/PerceivedComplexity:
+
+Metrics/ClassLength:
   Exclude:
-    - 'lib/puppet/provider/uri_parsing.rb'
+    - 'lib/puppet/provider/oneview_resource.rb'
+
 Metrics/CyclomaticComplexity:
   Max: 10
```

Comment:
Consider reduce the code complexity to 8, since no code changes are required at this point.